src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1210,17 +1214,7 @@ int fx_muxer_t::parse_args_and_execute(
         return InvalidArgFailure;
     }
 
-    if (cur_i != 1)
-    {
-        vec_argv.reserve(argc - cur_i + 1); // +1 for dotnet
-        vec_argv.push_back(argv[0]);
-        vec_argv.insert(vec_argv.end(), argv + cur_i, argv + argc);
-        new_argv = vec_argv.data();
-        new_argc = vec_argv.size();
-    }
-
-    // Transform dotnet [exec] [--additionalprobingpath path] [--depsfile file] [dll] [args] -> dotnet [dll] [args]
-    return read_config_and_execute(own_dir, app_candidate, opts, new_argc, new_argv, mode);
+    return 1;
```

Comment:
I thought the convention was to return `0` on success.

Reply:
Yes. Thanks